src/main/js/dynamic/loaded/users/UserManagement.js

```diff
@@ -12,15 +12,18 @@ class UserManagement extends Component {
     constructor(props) {
         super(props);
 
-        const descriptor = DescriptorUtilities.findDescriptorByNameAndContext(this.props.descriptors, DescriptorUtilities.DESCRIPTOR_NAME.COMPONENT_USERS, DescriptorUtilities.CONTEXT_TYPE.GLOBAL)[0];
-        this.state = { descriptor };
     }
 
     render() {
-        const { descriptor } = this.state;
-        const { label, description } = this.props;
-        const canCreate = DescriptorUtilities.isOperationAssigned(descriptor, DescriptorUtilities.OPERATIONS.CREATE);
-        const canDelete = DescriptorUtilities.isOperationAssigned(descriptor, DescriptorUtilities.OPERATIONS.DELETE);
+        const { label, description, descriptors } = this.props;
+        const descriptorList = DescriptorUtilities.findDescriptorByNameAndContext(descriptors, DescriptorUtilities.DESCRIPTOR_NAME.COMPONENT_USERS, DescriptorUtilities.CONTEXT_TYPE.GLOBAL)
```

Comment:
Should we have a findSingleDescriptorByNameAndContext or modify findDescriptorByNameAndContext to perform encapsulate this code into the method so that we don't have to copy and paste the if check?

Add this code to the findDescriptorByNameAndContext or create a new method:
`
let foundDescriptor = null;
if(descriptorList && descriptorList.length > 0) {
  foundDescriptor = descriptorLIst[0];
}
`